iphone/Maps/UI/EditBookmark/BookmarkUIUtils.swift

```diff
@@ -20,6 +23,10 @@ extension BookmarkColor {
   func image(_ iconName: String) -> UIImage {
     circleImageForColor(color, frameSize: 22, iconName: iconName)
   }
+
+  static func bookmarkColor(from color: UIColor) -> BookmarkColor? {
```

Comment:
Why is this method needed?

Reply:
When the `BookmarkColorViewController` is opened it can show the current selected color as `selected cell` (that passed with initializer):
```swift
  init(bookmarkColor: BookmarkColor) {
    self.bookmarkColor = bookmarkColor
    super.init(style: .grouped)
  }
```

It works only for bookmarks when color is predefined with specific enum `BookmarkColor`.

But when we create `BookmarkColorViewController` and use some color previously selected from UIColorPicker (for tracks) it can vary a lot and do match limited cases from `BookmarkColor`:
```swift
  // For iOS < 14.0 when the color cannot be mapped to predefined it will not be selected in the table view.
        let trackColor = BookmarkColor.bookmarkColor(from: trackColor) ?? .none
        let colorViewController = BookmarkColorViewController(bookmarkColor: trackColor)

```